R/run_ir_analysis.r

```diff
@@ -18,7 +18,7 @@ run_ir_analysis <- function(basicDir, analysisId, cohortDefinitions, dbms, conne
 
   query <- SqlRender::readSql(system.file("sql/sql_server", "delete_strata.sql", package = "IncidenceRateSkeleton"))
   query <- SqlRender::renderSql(query, tableQualifier = resultsDatabaseSchema, analysis_id = analysisId)$sql
-  query <- SqlRender::translateSql(query, targetDialect = connectionDetails$dbms)$sql
+  query <- SqlRender::translate(query, targetDialect = connectionDetails$dbms, oracleTempSchema=resultsDatabaseSchema)
```

Comment:
I'd like to suggest to add `tempDatabaseSchema` as an optional parameter with `resultsDatabaseSchema` as default